addons/web_editor/views/snippets.xml

```diff
@@ -180,7 +180,13 @@
                                 data-name="bg_image_opt"
                                 data-dependencies="bg_image_opt"/>
             </we-row>
-            <we-colorpicker string="  ⌙ Dynamic color" data-dynamic-color="true" data-name="dynamic_color_opt"/>
+            <we-row string=" ⌙ Dyn. Colors" title="Dynamic Colors" data-name="dynamic_color_opt"><!-- &emsp; -->
```

Comment:
To highlight the fact that the first blank in the string attribute is actually an &emsp;
I did this because this is how it was done in other parts of the code.
The &emsp; is needed to indent the "Dynamic colors" line (the replaced line contained 2 spaces which did not indent the rendered result)

Reply:
Good point, disregard this comment then :+1: